docs/attributes-registry/event.md

```diff
@@ -9,7 +9,23 @@ linkTitle: Events
 <!-- semconv registry.event(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  | Stability |
 |---|---|---|---|---|
-| `event.name` | string | Identifies the class / type of event. [1] | `browser.mouse.click`; `device.app.lifecycle` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `event.body` | string | The body of the event serialized into JSON string. [1] | `{"role":"user","content":"how to use Events API?"}`; `"plain string"` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
At one point we were considering an `event.data` attribute for the same purpose. As a result, the [field name in the API (JS)](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/api-events/src/types/Event.ts#L35) is currently named `data`. Whichever we choose, I think we should make the name consistent.

Reply:
makes sense, changed to `event.data`